docs/content/foundations/spacing/spacing-unit-tokens.mdx

```diff
@@ -0,0 +1,163 @@
+---
+browser_title: NCI Design System Spacing Unit Tokens
+page_title: Spacing Unit Tokens
+nav_label: Spacing Unit Tokens
+nav_order: 10
+template_type: utility
+
+design_tokens:
+  - heading: Spacing Tokens
+    description: |
+      The documentation below displays a target value in pixels, but the final output for tokens above 2px and below -2px is the eqivalent value in rem, which will vary depending on your project settings.
```

Comment:
I asked for the NOTE to go here below this paragraph. 

Add this here:
```
**Note:** Some units tokens come in two flavors: a **numeric token** (like 0.5 or -2) and a **string token** (like '05' or 'neg-2'). Utility classes use unquoted string tokens exclusively when they exist (like .margin-x-neg-2). Mixins and functions usually accept either, and we will note any exceptions.
```

Reply:
the note content breaks the utility page when placed in the description / doesn't handle markdown.
```YAMLException: unidentified alias "*Note:**" at line 12, column 10:
      **Note:** Some units tokens come in two f ... ```
      
We can move out of description and place above the content listed below to get the correct order, as the token yaml only has one description per object.